apps/ledger-live-desktop/tests/component/modal.component.ts

```diff
@@ -100,4 +100,13 @@ export class Modal extends Component {
       isVisible = await element.isVisible();
     }
   }
+
+  @step("Fill amount")
+  async fillAmount(amount: string) {
```

Comment:
I remember in previous PR you have removed this method from here and now you are adding it back?